x-pack/plugins/security_solution/public/timelines/components/duration/index.tsx

```diff
@@ -23,6 +23,7 @@ export const Duration = React.memo<{
 }>(({ contextId, eventId, fieldName, value }) => (
   <DefaultDraggable
     id={`duration-default-draggable-${contextId}-${eventId}-${fieldName}-${value}`}
+    // @ts-expect-error
```

Comment:
`name` is not defined in the scope. It uses deprecated global var `name` from `lib.dom.d.ts`. See all other changes in `security_solution` plugin code. @elastic/siem

Reply:
@restrry thanks so much for catching this.

Reply:
We are now tracking work to fix this. Thanks!